style.css

```diff
@@ -0,0 +1,94 @@
+.body {
+  max-width: 1920px;
+  min-width: 375px;
+  height: 100vh;
+  margin: 0 auto;
+  background: url('pic/iswanto-arif-448003-unsplash.png');
+  background-size: cover;
+}
+.header {
+  width: 100%;
+  padding: 89px 114px 0 200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+}
+.nav {
+  display: flex;
+  justify-content: center;
+}
+.nav__link:not(:last-child) {
+  margin: 0 34px 0 0;
+}
+.nav__link{
+  color: #fff;
+  text-decoration: none;
+  font-size: 22px;
+  font-family: 'Museo';
+}
+  .container{
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-start;
+  }
+.bali__title {
+  display: flex;
+  font-family: 'PTSerif';
+  font-style: normal;
+  font-weight: bold;
+  font-size: 280px;
+  line-height: 362px;
+  align-items: center;
+  justify-content: space-between;
+  color: #f8eee4;
+  opacity: 0.4;
+}
+.bali__subtitle{
+  font-family: 'PTSerif';
+  font-style: normal;
+  font-weight: bold;
+  font-size: 98px;
+  line-height: 127px;
+  display: flex;
+  align-items: center;
+  position: absolute;
+  left: 9.71%;
+  right: 60.98%;
+  top: 46.61%;
+  bottom: 41.28%;
+  color: #F8EEE4;
+}
+.bali__text{
+position: absolute;
+left: 5.49%;
+right: 56.82%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)